modules/Users/controller.php

```diff
@@ -110,9 +111,86 @@ protected function action_saveuserwizard()
         $_POST['receive_notifications'] = $current_user->receive_notifications;
         $_POST['user_theme'] = (string)SugarThemeRegistry::getDefault();
 
-        // save and redirect to new view
+        // Will pull in the users details from first page of the wizard
+        if (!empty($_POST['first_name'])) {
+            $current_user->first_name = ($_POST['first_name']);
+        }
+        if (!empty($_POST['last_name'])) {
+            $current_user->last_name = ($_POST['last_name']);
+        }
+        if (!empty($_POST['email1'])) {
+            $current_user->email1 = ($_POST['email1']);
+        }
+        if (!empty($_POST['phone_work'])) {
+            $current_user->phone_work = ($_POST['phone_work']);
+        }
+        if (!empty($_POST['phone_mobile'])) {
+            $current_user->phone_mobile = ($_POST['phone_mobile']);
+        }
+        if (!empty($_POST['messenger_type'])) {
+            $current_user->messenger_type = ($_POST['messenger_type']);
+        }
+        if (!empty($_POST['messenger_id'])) {
+            $current_user->messenger_id = ($_POST['messenger_id']);
+        }
+        if (!empty($_POST['address_street'])) {
+            $current_user->address_street = ($_POST['address_street']);
+        }
+        if (!empty($_POST['address_city'])) {
+            $current_user->address_city = ($_POST['address_city']);
+        }
+        if (!empty($_POST['address_state'])) {
+            $current_user->address_state = ($_POST['address_state']);
+        }
+        if (!empty($_POST['address_postalcode'])) {
+            $current_user->address_postalcode = ($_POST['address_postalcode']);
+        }
+        if (!empty($_POST['address_country'])) {
+            $current_user->address_country = ($_POST['address_country']);
+        }
+
+        // Saves User Details ONLY
+        $current_user->save();
+
+
+        // Will pull in the users Preferences from second page of the wizard
+        if (!empty($_POST['timezone'])){
```

Comment:
Small PSR-2 change :D
```suggestion
        if (!empty($_POST['timezone'])) {
```